features/Context/FieldTypeFormContext.php

```diff
@@ -144,23 +145,120 @@ public function theFieldDefinitionIsMarkedAsRequired()
      */
     public function theInputFieldsShouldBeFlaggedAsRequired()
     {
-        $inputNodeElements = $this->getSession()->getPage()->findAll(
+        $this->theFieldShouldBeFlaggedAsRequired('input');
+    }
+
+    /**
+     * @Given /^the field definition is set to single choice$/
```

Comment:
Well, then we remove one method and add two :) Though the new ones might be protected. I'm... https://xkcd.com/1378/